src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs

```diff
@@ -111,6 +162,53 @@ private void updateParameter(IParameter parameter, double value)
          parameter.IsFixedValue = false;
       }
 
+      public Individual CreateBaseIndividualForPopulation(Individual originalIndividual)
```

Comment:
@Yuri05 I am not sure about this. This looks very complicated.
 I just want one healthy individual with all the parameter changed from the user. I do not update parameters changed by the CKD parameter becaus ethat does not really make sense...does it? Say I change the volume of kidney to be bigger in my base individual and then use the individual to create CKD patients... this is kind of crazy

Reply:
to be discussed. while indeed it does not make much sense to change GFR_spec in the individual (s. the question above) - technically it still can be done. And for Volume_Kidney I could even imagine it might make sense.
So maybe in such a case we still additionaly multiply by the factor Value/DefaultValue? Not sure

Reply:
Technically it can still be done but at the end of the day, GFR_spec will be overwritten according to the selected algorithm. I think this is fair and it can be explained in the manual. And changing the volume of kidney does not make sense since it changes the eGFR which is an input of the user.

Reply:
I simplified the implementation by relying on the Parameter Updater